O/OCCT/bundled/patches/STEPConstruct_AP203Context.cxx.patch

```diff
@@ -0,0 +1,11 @@
+--- /dev/null
```

Comment:
It'd be good to upstream all generic patches, like this one I think?

Reply:
Do you mean to file an issue report with the OpenCASCADE Technology developers? I can try to do this but worry about response time.

Reply:
Pull requests, yes, but I wasn't suggesting for wait for them to accept those PRs before merging this one.  It's just that upstreaming the relevant patches reduces our maintenance burden.

Reply:
Totally agree, will follow up with the upstream repo.